### ISSUE TITLE
Question about r/static and r/user subreddits

### ISSUE THREAD

**Issue by fturco** [libertysoft3/reddit-html-archiver#11]
There are two subreddits called [r/static](https://www.reddit.com/r/static/) and [r/user](https://www.reddit.com/r/user/). What would happen if I fetched them and then regenerated the HTML files? Would reddit-html-archiver overwrite the `r/static` and `r/user` local directories?

**Comment by libertysoft3** [comment]
Yep, it would overwrite these local disk `r` directories and mess everything up. Not worth a fix imo.
